develer-staff/webinar_nanabianca
# add fps counter and use arrow key to inc/dec them

Release #4 

❓ What
Add an fps counter and the possibility to increment and decrement them using the arrow keys left and right


🔨 Test
- python main.py
- verify that the fps counter is readable
- verify that the arrows increment/decrement it
- check for strange behaviour
